permits/api.py

```diff
@@ -76,6 +85,9 @@ def get_queryset(self):
         if starts_at:
             start = datetime.datetime.strptime(starts_at, "%Y-%m-%d")
             base_filter &= Q(starts_at__gte=start)
+        if show_only_future:
+            start = datetime.datetime.now() - datetime.timedelta(days=7)
```

Comment:
Why not simply filter on events with end date >= today? If yes, also adapt comment at line 64)

Reply:
Actually, this is a mistake: we should filter event which ended before today!

Reply:
Fixed